azkaban-hadoop-security-plugin/src/main/java/azkaban/hive/HiveMetaStoreClientFactory.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
```

Comment:
Nice. Thanks for remembering to add the notice. I configured Idea to add it automatically. Did you do the same?